### ISSUE TITLE
Recognition of xOsmunimunda nothgen. nov.

### ISSUE THREAD

**Issue by choess** [pteridogroup/ppg#9]
### Name of taxon

xOsmunimunda

### Rank of taxon

Nothogenus

### Approximate number of species affected

2

### Description of change

This proposal recognizes the nothogenus x _Osmunimunda_ for hybrids between the genera _Osmunda_ and _Claytosmunda_ (Osmundaceae). It contains two species: x _Osmunimunda ruggii_, a hybrid between _Osmunda spectabilis_ and _Claytosmunda claytoniana_, and x _Osmunimunda nipponica_, a hybrid between _Osmunda japonica_ and _Claytosmunda claytoniana_ subsp. _pilosum_.

### Reason for change

Nothotaxa are planned to be recognized in PPG II. If _Claytosmunda_ and _Plenasium_ are separated from _Osmunda_ s.s. as genera, as in PPG I, these two species become intergeneric hybrids.

Both species are quite rare despite (at least in North America) the routine sympatry of the two parents, suggesting that these hybridization events are uncommon.

The first valid name for this nothogenus is x _Osmunimunda_ Xia Wan & Li Bing Zhang. (Wan & Zhang 2022). The earlier name x _Osmuntonia_ for this genus (Liu et al. 2020), though more euphonious, is invalid under Art. H.6.2 of the Shenzhen Code, prescribing the construction of nothogeneric names.

Alternatively, PPG II could revert to recognizing a broadly circumscribed but monophyletic _Osmunda_ which would render this nothogenus superfluous. However, _Claytosmunda_ and _Plenasium_ are believed to have diverged from _Osmunda_ s.s. in the Jurassic and Cretaceous, respectively, and they can be identified and distinguished in the fossil record. (Bomfleur et al. 2017).

Bomfleur B, Grimm GW, McLoughlin S. 2017. The fossil Osmundales (royal ferns)—a phylogenetic network analyses, revised taxonomy, and evolutionary classification of anatomically preserved trunks and rhizomes. PeerJ 5: 3433. [https://doi.org/10.7717/peerj.3433](https://doi.org/10.7717/peerj.3433)

Hong-Mei Liu, Eric Schuettpelz, Harald Schneider. 2020. Evaluating the status of fern and lycophyte nothotaxa in the context of the Pteridophyte Phylogeny Group classification (PPG I). Journal of Systematics and Evolution 58(6): 988-1002. [https://doi.org/10.1111/jse.12641](https://doi.org/10.1111/jse.12641)

Xia Wan, Li Bing Zhang. 2022. Three new fern names of hybrid genera and four new combinations in Athyriaceae, Osmundaceae, and Thelypteridaceae. Phytotaxa 575(1): 104-108. [https://doi.org/10.11646/phytotaxa.575.1.8](https://doi.org/10.11646/phytotaxa.575.1.8)

### Reference(s) for publication of the name

Xia Wan, Li Bing Zhang. 2022. Three new fern names of hybrid genera and four new combinations in Athyriaceae, Osmundaceae, and Thelypteridaceae. Phytotaxa 575(1): 104-108. [https://doi.org/10.11646/phytotaxa.575.1.8](https://doi.org/10.11646/phytotaxa.575.1.8)

### Code of Conduct

- [X] I agree to follow the PPG Code of Conduct

**Comment by susanfawcett** [comment]
@choess I agree with recognizing and listing all the nothogenera that have been supported by multiple lines of evidence. Others are highly suspect, and could be mentioned, but maybe not formally recognized (e.g., Glaphyropteridopsis x Pseudocyclosorus or xAsplenicystopteris).

**Comment by gyatskievych** [comment]
Osmunda sensu lato is widely recognized as a textbook case of evolutionary stasis, with very old fossils essentially indistinguishable from modern species. I believe that branch length by itself is insufficient to support generic splitting when there is insufficient genetic and morphological divergence. Thus, I would prefer to recognize Osmunda in the broad sense.

**Comment by susanfawcett** [comment]
@gyatskievych I think this is a good point, and it may have been better to vote on what genera we recognize in Osmundaceae before voting on the nothogenus. How would you circumscribe Osmunda?  Including Claytosmunda and Plenasium?  I would favor retaining Osmundastrum given its sister relationship to the rest of the family (Todea, Leptopteris, Osmunda s.l.).

**Comment by gyatskievych** [comment]
Actually, the sister relationship of Osmundastrum to the rest of the family was found to be based on a flawed analysis (there is a paper somewhere by Bonfleur and Renner that discusses this). Osmunda sensu lato is a monophyletic group. The most comprehensive classification is attached, which also considers the extensive fossil record. Bonfleur et al. (2017) did recognize the four genera formerly treated as Osmunda, but noted that Claytosmunda is paraphyletic if fossil species are included in the analysis.  Bomfleur et al. (2017), The fossil Osmundales (Royal Ferns)—a phylogenetic network analysis, revised taxonomy,
and evolutionary classification of anatomically preserved trunks and rhizomes. PeerJ 5:e3433; DOI 10.7717/peerj.3433
[Osmundales revision 2017.pdf](https://github.com/pteridogroup/ppg/files/11972379/Osmundales.revision.2017.pdf)

**Comment by choess** [comment]
I really like that paper! I'm not sure I find paraphyly objectionable in the fossil context, but while I've been acquiescent in the narrow circumscription of _Osmunda_ in part because that paper used the _Claytosmunda_ and _Plenasium_ segregates, I wouldn't object to a reversion to the older circumscription (inclusive of _Osmundastrum_ as well; [Bomfleur et al. 2015](https://link.springer.com/article/10.1186/s12862-015-0400-7) is the paper I think you were looking for, the discussion recapitulated in section "2.2.3.2 Subtribus Osmundinae" of the 2017 paper).

Broadening the circumscription of _Osmunda_ relative to PPG I should probably be a separate proposal cross-referenced to this one. I think I may go make a post in the Discussion section laying out my clerical to-do list so we can get a better feel for what proposals might interact with one another.

**Comment by susanfawcett** [comment]
@gyatskievych @choess  Thanks for sharing that reference!

**Comment by joelnitta** [comment]
> Broadening the circumscription of Osmunda relative to PPG I should probably be a separate proposal cross-referenced to this one. I think I may go make a post in the Discussion section laying out my clerical to-do list so we can get a better feel for what proposals might interact with one another.

@choess, I went ahead and started a discussion about circumscription of _Osmunda_: https://github.com/pteridogroup/ppg/discussions/24

**Comment by joelnitta** [comment]
This proposal was voted on during PPG Ballot 1 (voting period July 2023). A total of 45 votes were cast. There were 41 'Yes' votes (91.1%) and 4 'No' votes (8.9%). The proposal passes.
